catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DumpCommand.java

```diff
@@ -335,6 +365,75 @@ private void exportMetacard(File dumpLocation, Metacard metacard)
                 }
             }
         }
+
+        if (!ignoreContent) {
+            exportMetacardContent(metacard);
+        }
+    }
+
+    private void exportMetacardContent(Metacard metacard) {
+        Attribute attribute = metacard.getAttribute(Metacard.DERIVED_RESOURCE_URI);
+
+        if (attribute != null) {
+            List<Serializable> serializables = attribute.getValues();
+            serializables.forEach(serializable -> exportContentItem(serializable, metacard));
+        }
+
+        URI resourceUri = metacard.getResourceURI();
+        if (resourceUri == null) {
+            return;
```

Comment:
Nit: you should probably use the same null-checking pattern that you used for the derived resource uri:
```
if (resourceUri != null) {
    exportContentItem(resourceUri.toString(), metacard);
}
```